hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -1471,6 +1473,7 @@ public Properties build() {
       }
       if (null != commitTimeZone) {
         tableConfig.setValue(HoodieTableConfig.TIMELINE_TIMEZONE, commitTimeZone.toString());
+        HoodieInstantTimeGenerator.setCommitTimeZone(commitTimeZone);
```

Comment:
is there a way we can avoid expecting caller to call `HoodieInstantTimeGenerator.setCommitTimeZone` everytime. 

for eg, 
whenever we set tableConfig.setTimeZone, or whenever we build the tableConfig (while creating or updating), can we call `HoodieInstantTimeGenerator.setCommitTimeZone` internally. 
so that every caller does not need to worry about calling this api w/o miss.

Reply:
That's the intention. So the timezone is only being set in HoodieTableConfig and HoodieTableMetaClient. Nowhere in the write clients/table servce client we are setting the timezone.

Reply:
I am referring to L1322 adn 1476

Reply:
we are good here